CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [9.2.0](https://github.com/trussworks/react-uswds/compare/react-uswds-9.1.0...react-uswds-9.2.0) (2025-03-29)
+## [10.0.0](https://github.com/trussworks/react-uswds/compare/react-uswds-9.1.0...react-uswds-9.2.0) (2025-05-13)
```

Comment:
@NamibiaTorres for the changelog, you can actually just delete what was the 9.2.0 entry entirely.

What will happen is, after we merge this PR, release please will create a new PR for us with an updated changelog adding 10.0.0